deploy/basic/terraform/providers.tf

```diff
@@ -25,6 +25,9 @@ terraform {
       source  = "hashicorp/tls"
       version = "2.0.1" # Latest version as March 2021 = 3.1.0. Using 2.0.1 (April, 2020) for ORM compatibility
     }
+    acme = {
+      source = "vancluever/acme"
```

Comment:
Need to check is acme is valid for ORM and include the supported version. Also, need to update the .terraform.lck

Reply:
`vancluever/acme` is not a supported provider. https://docs.oracle.com/en-us/iaas/Content/ResourceManager/Concepts/providers.htm